basic/README.md

```diff
@@ -0,0 +1,32 @@
+# Basic - Free Portfolio Website Templates
+
+A basic template for you amazing programmers out there!
+
+![Basic - Screenshot](basic-screenshot.png "Basic - Screenshot")
+
+## Features
+
+✅ Fast
+
+✅ FREE
+
+✅ Mobile Responsive
+
+✅ SEO Friendly
+
+✅ Compatible with any web server
+
+✅ 0 Dependencies
+
+## How to use
+
+1. Close or manually download the code.
+2. Under the "plain" folder, edit the file `config.js`.
```

Comment:
Please double check all steps if they are applicable and if yes, make sure all content matches with the current template you have.

Reply:
Done guru